### DISCUSSION TITLE
How to set iOS and Android deployment target version?

### DISCUSSION THREAD

**Discussion by naresh33** [ionic-team/capacitor#4455] [Q&A]
Using capacitor 2.4.2 

I did update my capacitor.config.json with the following

 "ios": {
    "minVersion": "13.2"
  }
 
(or)

  "cordova": {
    "android": {
        "preferences": {
            "minSdkVersion": "28",
            "targetSdkVersion": "30"
        }
    },
    "ios": {
        "preferences": {
            "minVersion": "13.6"
        }
    }

**Comment by jcesarmobile** [comment]
If you have a `variables.gradle` file, you can change `minSdkVersion = 21` and `targetSdkVersion = 29` to the versions you want.

For iOS, change it from Xcode in the General info screen of your app's target
![Captura de pantalla 2021-04-12 a las 16 23 50](https://user-images.githubusercontent.com/1637892/114410200-7899f280-9bab-11eb-8828-15e7c9354867.png)

**Reply by naresh33** [reply]
Thanks for the reply.
How can we achieve setting target version without using Xcode or variables.gradle?
Any way that we can set the target version using a capacitor?

**Reply by jcesarmobile** [reply]
The "capacitor way" of doing this is by changing it from Xcode/Android Studio.

If you don't want to change it from Xcode/Android studio you can always use npm scripts to write those values for you, but Capacitor doesn't offer them.

**Reply by naresh33** [reply]
Very thanks.
